File: AtCoder/ABC153/ABC153B.cs
using System;
using System.Linq;

public class ABC153_B {
    private static void Main(string[] args) {
        var h = Read1LineAsIntArray()[0];
        var a = Read1LineAsIntArray();

        var answer = (a.Sum() >= h) ? "Yes" : "No";
        Console.WriteLine(answer);
    }
  
    private static int[] Read1LineAsIntArray() => Console.ReadLine().Trim().Split().Select(x => int.Parse(x)).ToArray();
}

File: AtCoder/ABC153/ABC153B.kt
fun main(args: Array<String>) {
    val h = readLine()!!.split(" ").map(String::toInt)[0]
    val a = readLine()!!.split(" ").map(String::toInt)

    val answer = if (a.sum() >= h) "Yes" else "No"
    println(answer)    
}
